game/squadrons/squadrondefloader.py

```diff
@@ -43,6 +43,8 @@ def load(self) -> dict[AircraftType, list[SquadronDef]]:
                         f"Not using squadron because {faction.name} cannot use "
                         f"{squadron_def.aircraft}: {path}"
                     )
+                    for aircraft in faction.aircrafts:
+                        logging.debug(f"factions {aircraft} does not match squadrons {squadron_def.aircraft}")
```

Comment:
whats the reason for this debug logging?

Reply:
literally just for debugging since I did not find the error in the faction file, only to find out they got removed due to a "wrong" mod config